src/library/columncache.cpp

```diff
@@ -154,3 +144,8 @@ void ColumnCache::slotSetKeySortOrder(double notationValue) {
     // Replace the existing sort order
     m_columnSortByIndex[keyColumnIndex] = keySortSQL;
 }
+
+const QString& ColumnCache::columnName(Column column) const {
+    DEBUG_ASSERT(static_cast<std::size_t>(column) < std::size(kColumnNameByEnum));
+    return kColumnNameByEnum[column];
+}
```

Comment:
Creating a QString form a QStringView is a deep copy. 
kColumnNameByEnum uses LIBRARYTABLE_ID and friends a QStringLiteral which is basically a constexpr, but does not have the keyword because of MSVC limitations.
So using a QStringView here for the sake of becoming a constexpr, will introduce more deep copies later where now implicit sharing is used.

Reply:
With a pointer it works.